data/service/service/standard.go

```diff
@@ -477,6 +483,18 @@ func (s *Standard) initializeDataSourceClient() error {
 	return nil
 }
 
+func (s *Standard) initializeSummaryClient() error {
+	s.Logger().Debug("Creating summary client")
+
+	clnt, err := dataSummaryClient.New(s.dataStore)
```

Comment:
What do you think about injecting the summarizer registry into the summary client (instead of the store, which is a fairly large dependency)? Something like:
```suggestion
	s.Logger().Debug("Creating summarizer registry")

	summarizerRegistry := dataSummary.New(
		s.dataStore.NewSummaryRepository().GetStore(),
		s.dataStore.NewDataRepository(),
	)

	s.Logger().Debug("Creating summary client")

	clnt, err := dataSummaryClient.New(summarizerRegistry)
```
Seems like this would be better for testing.